code/game/objects/effects/effect_system.dm

```diff
@@ -32,13 +32,6 @@ would spawn and follow the beaker, even if it is carried or thrown.
 		pixel_x += rand(-random_offset,random_offset)
 		pixel_y += rand(-random_offset,random_offset)
 
-
-
-/obj/effect/Destroy()
```

Comment:
Why did you remove this?

Reply:
![image](https://user-images.githubusercontent.com/49622619/143624469-e9894724-b567-4c63-b55e-72bf7a440331.png)